scripts/ci_setup.py

```diff
@@ -309,7 +317,11 @@ def __main(args: list) -> int:
         target_framework_moniker = dotnet.FrameworkAction.get_target_framework_moniker(framework)
         dotnet_version = dotnet.get_dotnet_version(target_framework_moniker, args.cli) if args.dotnet_versions == [] else args.dotnet_versions[0]
         commit_sha = dotnet.get_dotnet_sdk(target_framework_moniker, args.cli) if args.commit_sha is None else args.commit_sha
-        source_timestamp = dotnet.get_commit_date(target_framework_moniker, commit_sha, repo_url)
+        if(args.commit_time is not None):
+            parsed_timestamp = datetime.datetime.strptime(args.commit_time, '%Y-%m-%d %H:%M:%S %z').astimezone(datetime.timezone.utc)
```

Comment:
We should double check this, but I am pretty sure that we currently get the time in the pacific time zone.

Reply:
I was just wrong. We store the time in UTC